uplc-discharge.md

```diff
@@ -45,12 +45,15 @@ module UPLC-DISCHARGE
 
   rule dischargeTerm((lam X:UplcId T:Term), RHO:Map) => (lam X dischargeTerm(T, RHO))
 
+  rule dischargeTerm([ T1:Term T2:Term ], RHO:Map) => [dischargeTerm(T1, RHO) dischargeTerm(T2, RHO) ]
+
   rule dischargeTerm([ T1:Term (T2:Term TL:TermList) ], RHO:Map) =>
-       dischargeTermApp(TL, [dischargeTerm(T1, RHO) dischargeTerm(T2, RHO) ], RHO)
+       dischargeTermApp(TL, [ dischargeTerm(T1, RHO) dischargeTerm(T2, RHO) ], RHO)
```

Comment:
I think this can be written as
```
  rule dischargeTerm([ T1:Term TL:TermList ], RHO:Map) =>
       dischargeTermApp(TL, dischargeTerm(T1, RHO), RHO)
```

And the following two rules take care of applying the TermList.